src/shared/common/components/foi-actions.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Fragment, useContext } from 'react';
```

Comment:
I would prefer that tab to say 'Extensions' instead of 'Actions' and display a list of the extension already applied. All the other tabs refer to a type of data that the tab displays. But I know you are following the design passed to you.